src/components/shows/data.js

```diff
@@ -0,0 +1,331 @@
+import revue2019 from "../../assets/images/previous/2019.jpg"
```

Comment:
Maybe it's worth asking someone else to proofread this as well?

Reply:
Definitely, I have had another read through myself for the time being :D

Reply:
A few missing apostrophes which I commented on below but otherwise all looks great!